snapshot/snapshot.go

```diff
@@ -296,7 +294,7 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		if o.minLayerSize > 0 {
 			if strVal, ok := base.Labels[source.TargetSizeLabel]; ok {
 				if intVal, err := strconv.Atoi(strVal); err == nil {
```

Comment:
``strVal`` should be converted to ``int64`` via ``strconv.ParseInt(strVal, 10, 64)``